lib/pushmi_pullyu/swift_deposit.rb

```diff
@@ -0,0 +1,58 @@
+require 'openstack'
+
+class PushmiPullyu::SwiftDepositer
+
+  def initialize(connection: nil, container: nil)
```

Comment:
I would let ruby do it's job here. Instead of passing default arguments for nil, then checking if these arguments are nil and raising your exception. Just don't use default arguments instead?

Your code: 
```
2.4.0 :001 > PushmiPullyu::SwiftDepositer.new
RuntimeError: conection can not be nil
```
Rewritting this to be ```  def initialize(connection, container)``` now becomes:
```
2.4.0 :001 > PushmiPullyu::SwiftDepositer.new
ArgumentError: wrong number of arguments (given 0, expected 2)
```